blocks/search/search.js

```diff
@@ -0,0 +1,12 @@
+setTimeout(() => {
+  const formElement = document.getElementById('search');
+  const searchForm = document.querySelector('.search-filter');
+  if (searchForm) {
+    searchForm.removeAttribute('data-action');
+    searchForm.setAttribute('method', 'get');
+    // eslint-disable-next-line no-script-url
+    searchForm.setAttribute('action', 'http://google.com/search');
+  }
+  formElement.setAttribute('name', 'q');
+  formElement.setAttribute('autocomplete', 'off');
+}, 1500);
```

Comment:
Why do we need setTimout ?

Reply:
Because, we build the form at runtime from the parent component. I've used a timeout to add the event on input field after the DOM rendered.